Resources/ALCS1200A/Info.plist

```diff
@@ -63,6 +71,14 @@
 				<key>Path</key>
 				<string>PlatformsT.xml.zlib</string>
 			</dict>
+			<dict>
+				<key>Comment</key>
+				<string>toleda resources</string>
```

Comment:
Hello. Please fix comment if you are using Mirone resources.

Reply:
Oh! I didn't pay attention to this. Can you help me modify it? I am not familiar with this operation, sorry!